hasher-matcher-actioner/hmalib/lambdas/aws_secrets.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Ignore this file. It is already in [another pull request](https://github.com/facebook/ThreatExchange/pull/417) that will be merged first